ale/base/label_pds3.py

```diff
@@ -25,92 +15,202 @@ def label(self):
                 raise ValueError("{} is not a valid label".format(self._file))
         return self._label
 
-    @property
-    def _focal_plane_tempature(self):
-        return self.label['FOCAL_PLANE_TEMPERATURE'].value
 
     @property
     def instrument_id(self):
-        pass
+        """
+          Returns
+        -------
+        : str
+          Short name of the instrument
+        """
+        return self.label['INSTRUMENT_ID']
+
+
+    @property
+    def instrument_name(self):
+        """
+          Returns
+        -------
+        : str
+          Full name of the instrument
+        """
+        return self.label['INSTRUMENT_NAME']
+
+
+    @property
+    def instrument_host_id(self):
+        """
+          Returns
+        -------
+        : str
+          Short name of the instrument host
+        """
+        return self.label['INSTRUMENT_HOST_ID']
+
+
+    @property
+    def instrument_host_name(self):
+        """
+          Returns
+        -------
+        : str
+          Full name of the instrument host
+        """
+        return self.label['INSTRUMENT_HOST_NAME']
+
 
     @property
     def start_time(self):
```

Comment:
Just to be consistent, in my PR the time type is embedded in the name. Maybe change this to `utc_start_time`.